src/taxman.py

```diff
@@ -126,89 +126,150 @@ def evaluate_sell(op: transaction.Operation) -> Optional[transaction.TaxEvent]:
+                    cost = self.price_data.get_cost(op)
+                    price = self.price_data.get_price(
+                        op.platform,
+                        op.coin,
+                        op.utc_time,
+                        config.FIAT
+                    )
+                    remark = (
+                        f"Kosten {cost} {config.FIAT}, "
+                        f"Preis {price} {config.FIAT}/{op.coin}"
+                    )
+                    tx = transaction.TaxEvent(
+                        taxation_type,
+                        decimal.Decimal(),
+                        op,
+                        False,
+                        decimal.Decimal(),
+                        decimal.Decimal(),
+                        remark
+                    )
             elif isinstance(op, transaction.Sell):
+                if misc.is_fiat(op.coin):
+                    continue
                 if tx_ := evaluate_sell(op):
-                    self.tax_events.append(tx_)
+                    tx = tx_
+                else:
+                    taxation_type = (
+                        "Verkauf "
+                        "(außerhalb des Steuerjahres oder steuerfrei)"
+                    )
+                    tx = transaction.TaxEvent(
+                        taxation_type,
+                        decimal.Decimal(),
+                        op,
+                        False
+                    )
             elif isinstance(
                 op, (transaction.CoinLendInterest, transaction.StakingInterest)
             ):
                 balance.put(op)
+                if misc.is_fiat(coin):
+                    assert not isinstance(
+                        op, transaction.StakingInterest
+                    ), "You can not stake fiat currencies."
+                    taxation_type = "Einkünfte aus Kapitalvermögen"
+                else:
+                    taxation_type = "Einkünfte aus sonstigen Leistungen"
                 if self.in_tax_year(op):
-                    if misc.is_fiat(coin):
-                        assert not isinstance(
-                            op, transaction.StakingInterest
-                        ), "You can not stake fiat currencies."
-                        taxation_type = "Einkünfte aus Kapitalvermögen"
-                    else:
-                        taxation_type = "Einkünfte aus sonstigen Leistungen"
-                    taxed_gain = self.price_data.get_cost(op)
-                    tx = transaction.TaxEvent(taxation_type, taxed_gain, op)
-                    self.tax_events.append(tx)
+                    is_taxable = True
+                else:
+                    taxation_type += " außerhalb des Steuerjahres"
+                    is_taxable = False
+                taxed_gain = self.price_data.get_cost(op)
+                tx = transaction.TaxEvent(taxation_type, taxed_gain, op, is_taxable)
             elif isinstance(op, transaction.Airdrop):
```

Comment:
According to https://www.winheller.com/bankrecht-finanzrecht/bitcointrading/bitcoinundsteuer/besteuerung-airdrops.html and the publication by the BMF, Airdrops may be considered as "sonstige Einkünfte" in the future.

Reply:
Please make a new PR for this adjustment. It might become to complicated with we do everything in this branch :)

Reply:
Done, https://github.com/provinzio/CoinTaxman/issues/104